js/index.js

```diff
@@ -59,7 +60,7 @@ export default class App extends Component {
 
   render() {
```

Comment:
this.state -> loadingUser should come from the redux store, it should not be in the component state. Please make the change.

Reply:
just use the userLoading(true/false).

Reply:
Mackenzie and I tried to resolve this but could not make it work.